File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/javascript/cwe-863/code_before1.js
// Source: Row 15 in ./dataset/CVEfixes/Analysis/results/JavaScript/df_javascript_cwe_863.xlsx

_size ({ id, token }, done) {
    return this.client
      .get(`https://graph.facebook.com/${id}`)
      .qs({ fields: 'images' })
      .auth(token)
      .request((err, resp, body) => {
        if (err || resp.statusCode !== 200) {
          err = this._error(err, resp)
          logger.error(err, 'provider.facebook.size.error')
          return done(err)
        }

        getURLMeta(this._getMediaUrl(body))
          .then(({ size }) => done(null, size))
          .catch((err2) => {
            logger.error(err2, 'provider.facebook.size.error')
            done(err2)
          })
      })
  }

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/python/cwe-352/code_before1.py
# Source: Row 28 in ./dataset/CVEfixes/Analysis/results/Python/df_python_cwe_352.xlsx

def test_get_delete_dataobj(test_app, client: FlaskClient, note_fixture):
    response = client.post("/dataobj/delete/1")
    assert response.status_code == 302